.eslintrc.json

```diff
@@ -53,7 +53,7 @@
 			}
 		],
 		"strict": "off",
-		"space-before-function-paren": ["error", "never"],
+		"space-before-function-paren": "off",
```

Comment:
@nazarhussain It's conflicting with prettier. We already have lots of overrides in eslintrc.json file I would say let's keep it like this for the moment. I believe since we moved to node 8, we can refactor lots of eslint rules in near future.